packages/service-configuration/service_configuration_server.js

```diff
@@ -28,3 +30,17 @@ try {
     );
     throw err;
 }
+
+Meteor.startup(() => {
+    if (!Meteor.settings?.packages?.['service-configuration']) return;
+    const settings = Meteor.settings?.packages?.['service-configuration']
```

Comment:
You could get the property first and then return or proceed

```suggestion
    const settings = Meteor.settings?.packages?.['service-configuration'];
    if (!settings) {
      return;
    }
    // proceed
```